commands.go

```diff
@@ -74,6 +75,9 @@ var (
 		{"colour", colorCMD, "???", "This is an alias of color"}, // appease the british
 		{":q", exitCMD, "", "This is an alias of exit"},          // appease the Vim user
 		{":wq", exitCMD, "", "This is an alias of exit"},         // appease the Vim user, that wants to save
+		{"finger", lsCMD, "???", "This is an alias of id"}, // appease the Linux / Unix user
```

Comment:
did you mean to make this an alias to idCMD?

Reply:
yes